pkg/types/defaults/installconfig.go

```diff
@@ -68,6 +68,11 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 	c.ControlPlane.Name = "master"
 	SetMachinePoolDefaults(c.ControlPlane, c.Platform.Name())
 
+	if c.Arbiter != nil {
+		c.Arbiter.Name = "arbiter"
+		SetMachinePoolDefaults(c.Arbiter, c.Platform.Name())
```

Comment:
Can you also modify [SetMachinePoolDefaults](https://github.com/openshift/installer/blob/main/pkg/types/defaults/machinepools.go#L11) to make sure that the replica count is set to 0 for Arbiters. We can let the validation catch 0 so the user makes an explicit decision if they forget to add the value.